mario.js

```diff
@@ -275,6 +275,17 @@ class Mario {
                                 that.game.camera.loadLevel(bonusLevelOne, 2.5 * PARAMS.BLOCKWIDTH, 0 * PARAMS.BLOCKWIDTH);
                             }
                         }
+                        if (entity instanceof Lift && that.lastBB.bottom <= entity.BB.top + PARAMS.SCALE * 3) {
```

Comment:
fyi, irrc mario can instantly die if hes underneith a platform as it pushs himinto the ground.

as this has to do with more "advanced" bounding-box/collision detection (involves 3+ entity's), it should probably be a seperate issue